resolver-dns/src/test/java/io/netty/resolver/dns/DnsNameResolverTest.java

```diff
@@ -3734,7 +3735,7 @@ public Set<ResourceRecord> getRecords(QuestionRecord question) {
 
     @Test
     public void testCNAMEOnlyTriedOnAddressLookups() throws Exception {
```

Comment:
@tkountis can you also add a test case which would show that we not try a cname lookup if the property is false ?